tests/test_app.py

```diff
@@ -167,9 +168,27 @@ def test_session_stats_create_without_session_record(self):
         self.assertIsNotNone(session.client_updated_at)
         self.assertEqual(auth['public_address'], session.consumer_id)
         self.assertEqual('8.8.8.X', session.client_ip)
-        self.assertEqual('US', session.client_country)
+        self.assertEqual('country', session.client_country)
         self.assertEqual('0x1', session.node_key)
 
+    def test_session_stats_without_consumer_country(self):
```

Comment:
Shouldn't we identify that this test is without session record, as in previous test?